x-pack/plugins/infra/server/routes/metrics_explorer_views/README.md

```diff
@@ -0,0 +1,350 @@
+
+```json
+GET /api/infra/metrics_explorer_views/927ad6a0-da0c-11ed-9487-41e9b90f96b9
+
+Status code: 200
+
+{
+  "data": {
+    "id": "927ad6a0-da0c-11ed-9487-41e9b90f96b9",
+    "version": "WzQwMiwxXQ==",
+    "updatedAt": 1681398305034,
+    "attributes": {
+      "name": "Ad-hoc",
+      "isDefault": true,
+      "isStatic": false,
+      "metric": {
+        "type": "cpu"
+      },
+      "sort": {
+        "by": "name",
+        "direction": "desc"
+      },
+      "groupBy": [],
+      "nodeType": "host",
+      "view": "map",
+      "customOptions": [],
+      "customMetrics": [],
+      "boundsOverride": {
+        "max": 1,
+        "min": 0
+      },
+      "autoBounds": true,
+      "accountId": "",
+      "region": "",
+      "autoReload": false,
+      "filterQuery": {
+        "expression": "",
+        "kind": "kuery"
+      },
+      "legend": {
+        "palette": "cool",
+        "reverseColors": false,
+        "steps": 10
+      },
+      "timelineOpen": false
+    }
+  }
+}
+```
+
+```json
+GET /api/infra/metrics_explorer_views/random-id
+
+Status code: 404
+
+{
+  "statusCode": 404,
+  "error": "Not Found",
+  "message": "Saved object [metrics-explorer-view/random-id] not found"
+}
+```
+
+## Create one: `POST /api/infra/metrics_explorer_views`
+
+Creates a new metrics explorer view.
+
+### Request
+
+- **Method**: POST
+- **Path**: /api/infra/metrics_explorer_views
+- **Request body**:
+  ```json
+  {
+    "attributes": {
+      "name": "View name",
+      "metric": {
+        "type": "cpu"
+      },
+      "sort": {
+        "by": "name",
+        "direction": "desc"
+      },
+      //...
+    }
+  }
+  ```
+
+### Response
+
+```json
+POST /api/infra/metrics_explorer_views
+
+Status code: 201
+
+{
+  "data": {
+    "id": "927ad6a0-da0c-11ed-9487-41e9b90f96b9",
+    "version": "WzQwMiwxXQ==",
+    "updatedAt": 1681398305034,
+    "attributes": {
+      "name": "View name",
+      "isDefault": false,
+      "isStatic": false,
+      "metric": {
+        "type": "cpu"
+      },
+      "sort": {
+        "by": "name",
+        "direction": "desc"
+      },
+      "groupBy": [],
+      "nodeType": "host",
+      "view": "map",
+      "customOptions": [],
+      "customMetrics": [],
+      "boundsOverride": {
+        "max": 1,
+        "min": 0
+      },
+      "autoBounds": true,
+      "accountId": "",
+      "region": "",
+      "autoReload": false,
+      "filterQuery": {
+        "expression": "",
+        "kind": "kuery"
+      },
+      "legend": {
+        "palette": "cool",
+        "reverseColors": false,
+        "steps": 10
+      },
+      "timelineOpen": false
+    }
+  }
+}
+```
+
+Send in the payload a `name` attribute already held by another view:
+```json
+POST /api/infra/metrics_explorer_views
+
+Status code: 409
+
+{
+  "statusCode": 409,
+  "error": "Conflict",
+  "message": "A view with that name already exists."
+}
+```
+
+## Update one: `PUT /api/infra/metrics_explorer_views/{metricsExplorerViewId}`
```

Comment:
Does it make sense to document the 400 returned when `PUT /api/infra/metrics_explorer_views/0` is called?